active_model_command.gemspec

```diff
@@ -28,5 +28,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 3.0"
   spec.add_development_dependency "rspec_junit_formatter", "~> 0.4"
 
-  spec.add_runtime_dependency "activemodel", ">= 2.2.0"
```

Comment:
I added the gem in order to stablish an in memory database connection so `ActiveRecord::Base.transaction` won't raise an `ActiveRecord::ConnectionNotEstablished` exception